vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1522,7 +1573,7 @@ def _dummy_run(
                     for k, v in self.intermediate_tensors.items()
                 })
 
-            with set_forward_context(None,
+            with set_forward_context(attn_metadata,
```

Comment:
Considering that `self.maybe_setup_kv_connector(scheduler_output)` is not executed here, in the Full Cuda Graph scenario, the sequence `unified_attention_with_output` -> `maybe_save_kv_layer_to_connector` -> `connector.save_kv_layer()` will cause the connector to read uninitialized metadata. 

https://github.com/LMCache/LMCache/blob/680fbdf84e2ee1040bf4e084d43c9155a91b8d5c/lmcache/integration/vllm/vllm_v1_adapter.py#L609-L610

Therefore, Full Cuda Graph should be incompatible with kvconnector?